app/src/main/res/layout/cast_card.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="88dp"
+    android:paddingHorizontal="4dp"
+    android:layout_height="118dp"
+    android:orientation="vertical">
+
+    <ImageView
+        android:id="@+id/castPhoto"
+        android:layout_width="match_parent"
+        android:layout_height="80dp"
+        android:scaleType="fitXY"
+        app:srcCompat="@drawable/mock_picture_bieber" />
+
+    <TextView
+        android:id="@+id/castName"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center_vertical"
+        android:layout_marginTop="6dp"
+        android:layout_marginEnd="10dp"
+        android:textColor="@color/white_gray"
+        android:textSize="12dp"
```

Comment:
Use SP instead

Reply:
I use sp only in long paragraphs. In small cards with fixed size it can break all design (e.g. if user select large system font)